final_project.ipynb

```diff
@@ -119,7 +286,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [],
    "source": [
```

Comment:
for `device_num`, assign this instead para applicable sa different devices

`device_num = torch.cuda.get_device_name(0)`